jujugui/static/gui/src/app/components/lightbox/_lightbox.scss

```diff
@@ -21,35 +21,117 @@
     }
   }
 
-  &__content {
+  &__wrapper {
     position: absolute;
     top: 10%;
     left: 10%;
     bottom: 15%;
     right: 10%;
     text-align: center;
+  }
 
-    &-image {
-      background: $color-light;
-      border-radius: 2px;
-      padding: 2rem;
-      width: 100%;
-      height: 100%;
+  &__content {
+    background: $color-light;
+    border-radius: 2px;
+    padding: 2rem;
+    width: 100%;
+    height: 100%;
+    position: relative;
 
-      * {
-        max-height: 100%;
-        max-width: 100%;
+    * {
+      max-height: 100%;
+      max-width: 100%;
+    }
+  }
+
+  &__caption {
+    position: absolute;
+    bottom: -2rem;
+    left: 0;
+    right: 0;
+    text-align: center;
+    color: $white;
+  }
+
+  &__slide {
+    display: none;
+
+    &.is-active {
+      display: block;
+    }
+  }
+
+  &__navigation {
+    position: absolute;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    top: 0;
+    // This is to put the content above the navigation
+    z-index: -1;
+
+    &-previous,
+    &-next {
+      position: absolute;
+      border: 0;
+      background: $white;
+      width: 3rem;
+      height: 3rem;
+      display: block;
+      border-radius: 3rem;
+      overflow: hidden;
+      text-indent: -9999px;
+      margin: 0;
+      padding: 0;
+      text-align: center;
+      top: 50%;
+      margin-top: -1.5rem;
+      background: {
+        color: $white;
+        image: url('/static/gui/build/app/assets/svgs/chevron_down_16.svg');
```

Comment:
this won't work as the static path can change. You'll have to implement this in the component with the supplied path.